charts/jellyseerr-chart/README.md

```diff
@@ -64,7 +63,17 @@ Kubernetes: `>=1.23.0-0`
 | serviceAccount.automount | bool | `true` | Automatically mount a ServiceAccount's API credentials? |
 | serviceAccount.create | bool | `true` | Specifies whether a service account should be created |
 | serviceAccount.name | string | `""` | If not set and create is true, a name is generated using the fullname template |
-| strategy | object | `{"type":"Recreate"}` | Deployment strategy |
 | tolerations | list | `[]` |  |
-| volumeMounts | list | `[]` | Additional volumeMounts on the output Deployment definition. |
-| volumes | list | `[]` | Additional volumes on the output Deployment definition. |
+| volumeMounts | list | `[]` | Additional volumeMounts on the output StatefulSet definition. |
+| volumes | list | `[]` | Additional volumes on the output StatefulSet definition. |
+
+## Update Notes
+
+### Updating to 2.7.0
+
+Jellyseerr is a stateful application and it is not designed to have multiple replicas. In version 2.7.0 we address this by:
+
+- replacing `Deployment` with `StatefulSet`
+- removing `replicaCount` value
+
+If `replicaCount` value was used - remove it. Helm update should work fine after that.
```

Comment:
This note must be added to the `README.md.gotmpl` file, otherwise it will be lost during the next `helm-docs` run. Let's add it before `{{ template "chart.valuesSection" . }}`.

Reply:
Thanks for noticing this, fixed.

Reply:
Don't forget to run helm-docs the README is missing the update notes section from the template

Reply:
Done.